package.json

```diff
@@ -1,20 +1,21 @@
 {
-  "name": "@optbot/template",
-  "version": "0.0.2",
-  "description": "template for starting a new package",
-  "main": "lib/start",
+  "name": "@optbot/eqpuller",
+  "version": "0.0.0",
+  "description": "maintain a list of active equities",
+  "main": "lib/norequire",
```

Comment:
These are actually different. One is the target for `require()` calls, and the other is a full command, which can actually be anything that executes. However, I notice that, while everything here works with or without the file extension, `npm` itself includes it. I'm also changing template accordingly.